main.tf

```diff
@@ -0,0 +1,45 @@
+data "template_file" "body" {
+  template = "${var.body}"
+
+  vars = "${var.vars}"
+}
+
+data "template_file" "subject" {
+  template = "${var.subject}"
+
+  vars = "${var.vars}"
+}
+
+locals {
+  body    = "${data.template_file.body.rendered}"
+  subject = "${data.template_file.subject.rendered}"
+  command = "${var.mail_command} ${join(" ", var.to)}"
```

Comment:
is this specific to this tool https://github.com/wrouesnel/emailcli?

Reply:
Pretty much. It can be overridden, but it's out of scope to support generic tools. The reason `mail_command` is an argument is that it might be installed to non-system paths (e.g. `/home/user/bin/email`)